src/store/index.js

```diff
@@ -18,6 +20,15 @@ export default new Vuex.Store({
     },
     updateGeolocation (state, geolocation) {
       state.geolocation = geolocation
+    },
+    storeEmail (state, email) {
```

Comment:
Thanks for adding these mutations. That gives us flexibility if we ever want to do this functionality elsewhere